src/Spotify.js

```diff
@@ -0,0 +1,20 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    host === 'open.spotify.com' &&
+    (pathname.includes('/album/') || pathname.includes('/track/'))
+  );
+};
+
+export const getHTML = url => {
+  const isTrack = url.includes('/track/');
+
+  const replaceKey = isTrack ? '/track/' : '/album/';
+  const iframeUrl = url.replace(replaceKey, `/embed${replaceKey}`);
```

Comment:
I think we should extract this into a separate method (`getSpotifyIFrameSrc`), like we do in the YouTube transformer

https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/da007b20ef4ea9c5efd4abd3d3d3e73556f15283/src/YouTube.js#L50-L54